py/bitbox02/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased]
 
+# 8.0.0
```

Comment:
Also bump the actual package version please: https://github.com/BitBoxSwiss/bitbox02-firmware/blob/878ea00dc91d2675de378bba24062f188cf104b8/py/bitbox02/bitbox02/bitbox02/__init__.py#L19

Reply:
We can take a look at best practices. I think there also are some changelog tooling, maybe that helps.